src/HashStore.hs

```diff
@@ -1,7 +1,15 @@
 module HashStore
-       ( someFunc
+       ( hashContent
+       , hashFile
        ) where
 
-someFunc :: IO ()
-someFunc = putStrLn ("someFunc" :: String)
+import Crypto.Hash.BLAKE2.BLAKE2b (hash)
+import Data.ByteString.Char8 (ByteString, unpack)
 
+-- | Perform 'ByteString' hashing.
+hashContent :: ByteString -> ByteString
+hashContent = hash 64 mempty
```

Comment:
I think it's better to not export `hashContent` function for optimization purposes. And add `{-# INLINE` to it just in case.